packages/truffle-compile/package.json

```diff
@@ -8,6 +8,7 @@
     "colors": "^1.1.2",
     "debug": "^3.1.0",
     "find-cache-dir": "^1.0.0",
+    "ora": "^3.0.0",
```

Comment:
Can you also upgrade `truffle-reporters`'s dependency? Or just use the same one that package is using (`^2.1.0`)